write_locations.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+"""
+Write one or more sets of publication location information to Thoth.
+Input: path to file containing location information, one location per line,
+containing publication ID, location platform, landing page and full text URL,
+separated by spaces.
+Requires: Thoth credentials as THOTH_EMAIL and THOTH_PWD env vars.
+"""
+
+# Third-party package already included in thoth-dissemination/requirements.txt
+from thothlibrary import ThothClient, ThothError
+from os import environ
+import sys
+
+
+def write_thoth_location(publication_id, location_platform, landing_page,
+                         full_text_url):
+    thoth = ThothClient()
+    username = environ.get('THOTH_EMAIL')
+    password = environ.get('THOTH_PWD')
+    if username is None:
```

Comment:
If you access the `environ` dictionary directly (`environ['THOTH']`) you don't need to check for unset variables – you'll get a `KeyError` exception